packages/scene-composer/src/components/toolbars/floatingToolbar/items/AddObjectMenu.tsx

```diff
@@ -197,7 +199,7 @@ export const AddObjectMenu = ({ canvasHeight, toolbarOrientation }: AddObjectMen
     });
   }, [selectedSceneNodeRef]);
 
-  const handleAddViewCamera = useCallback(() => {
```

Comment:
Tested adding a camera view when first loading the scene, and it fails. Can still add a camera view after adding something else.
Need to remove this `useCallback` that I introduced with the feature flag removal.